### ISSUE TITLE
Reproducible out of memory crash

### ISSUE THREAD

**Issue by uramer** [nickwalton/AIDungeon#29]
Describe the current behavior:
Out of memory crash. To reproduce: load 95c2fb00-1906-11ea-a04f-0242ac1c0002 and type "Try to cast the spell written on the wall"

Describe the expected behavior:
No crash.

The web browser you are using (Chrome, Firefox, Safari, etc.):
Desktop Chrome Version 79.0.3945.74 (Official Build) beta (64-bit)

**Comment by dyc3** [comment]
I'm unable to reproduce this in google colab. Are you running it locally?

**Comment by uramer** [comment]
No, I was running it in collab. Haven't checked since I've reported it
though, might have time tomorrow.
